io/io/src/TKey.cxx

```diff
@@ -884,6 +898,20 @@ TObject *TKey::ReadObjWithBuffer(char *bufferRead)
       return (TObject*)ReadObjectAny(0);
    }
 
+   struct CleanUp {
```

Comment:
Can you move this struct into an unnamed namespace outside this function, given that there's 100% code dupe with the `CleanUp` above and below?